GirafRest/Controllers/PictogramController.cs

```diff
@@ -346,13 +346,12 @@ public async Task<ActionResult> SetPictogramImage(long id)
 
 
 
-                pictogram.ImageHash = image.GetHashCode().ToString();
+                pictogram.ImageHash = _giraf.GetHash(image);
```

Comment:
We change in pictogram controller. This should be tested if it works.

Reply:
Gethash() is tested, and pictogramcontroller should have tests from when it was created